src/main/java/by/exadel/internship/controller/InternshipController.java

```diff
@@ -71,4 +72,12 @@ public UserInternshipDTO restoreInternship(@PathVariable UUID internshipId) {
     public UserInternshipDTO saveInternship(@RequestBody UserInternshipDTO internshipDTO) {
         return internshipService.saveInternship(internshipDTO);
     }
+
+    @SuperAdminAccessControl
+    @PostMapping("/addUser")
```

Comment:
I guess if `internshipId` is mandatory variable it'll be better to move it to path variable.

```suggestion
    @PostMapping("/{internshipId}/addUser")
```

Reply:
done